src/js/setup/dataexplorer.js

```diff
@@ -1,16 +1,17 @@
-import {loadMenu, showNoData, DataMenu} from '../elements/menu';
+import { LoadMenu, showNoData, DataMapperMenu } from '../elements/menu';
 
 export function configureDataExplorerEvents(controller, dataMapperMenu) {
-    controller.bubbleEvent(dataMapperMenu, 'data_mapper_menu.nodata')
-    controller.on('profile.loaded', payload => {
-        if ($.isEmptyObject(payload.payload.geometries.children)) {
-            //no children -- show no-data chip
-            dataMenu.showNoData();
-        } else {
-            const profileData = payload.payload.profile.profileData;
-            loadMenu(dataMapperMenu, profileData, payload => {
-                controller.onSubIndicatorClick(payload)
-            })
-        }
-    })
+  controller.bubbleEvent(dataMapperMenu, 'data_mapper_menu.nodata')
+  controller.on('profile.loaded', payload => {
+    const loadMenu = new DataMapperMenu(dataMapperMenu);
```

Comment:
could you change `loadMenu` to `dataMapperMenu`

Reply:
changed to `dataMenu` we have a lot of variables already named dataMapperMenu

Example: `new DataMapperMenu(dataMapperMenu);`